styles/components/JobDetails.module.scss

```diff
@@ -0,0 +1,21 @@
+@import '../globals.scss';
+
+.JobDetails {
+  .subtitle {
+    margin-top: -16px;
+    color: lighten($text-color, 30);
+    font-size: 14px;
+  }
+  .howToApply {
+    font-size: 12px;
+    background-color: lighten($primary, 30);
+    border-radius: 6px;
+    border: 3px solid lighten($primary, 20);
+    h5 {
```

Comment:
Although is not bad, switching this to a class selector instead of the node can prevent you from maintenance updates.

Reply:
Like this?

```
.information-box {
  font-size: 12px;
  background-color: lighten($primary, 30);
  border-radius: 6px;
  border: 3px solid lighten($primary, 20);
}
```

Reply:
yes, perfect. We can go deeper on BEM later on.